solverMULES/alphaEqn.H

```diff
@@ -101,7 +101,7 @@ License
         allLambda = 1.0;
 
         // Solve for alpha1
-        solve(fvm::ddt(alpha) + fvc::div(alphaPhi1));
+    solve(fvm::ddt(alpha) + fvc::div(alphaPhi1));
```

Comment:
Please revert this modification : indent correctly the solve command as in the original file